entry_types/scrolled/package/src/frontend/useEventContextData.js

```diff
@@ -0,0 +1,39 @@
+import React, {createContext, useContext, useMemo} from 'react';
+import {useEntryStructure} from '../entryState';
+
+export function getEventObject(sectionIndex, chapterIndex, entryStructure){
```

Comment:
I think first we should identify what is required. 
If chapterIndex is required then getEventObject interface 
is not more important than requirement may be still it can be adjusted.
if chapterIndex is not required than it can simply be removed from getEventObject .

Reply:
I'd say there is more to be gained by just always providing an event object with the same form, because there isn't a domain-based or pragmatic reason to omit chapter index from the page object in any case.

If I imagine a future developer who would like to do something with the page object, they might start cursing when they notice that it has the chapter index only in the case of media events.

I agree with you that this version of `getEventObject` does its job in terms of requirements, but I think we could easily do better both in terms of simplifying the function signature and in terms of providing consistent output.  

The latter is important especially because we don't have TypeScript or similar, and because usage of the `page` object happens in other modules. It could be really time-consuming to have to reconstruct the arcane reason why chapter index is only sometimes included in the page object